README.md

```diff
@@ -40,12 +40,20 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 - [ ] Explain the difference between `Relational Databases` and `SQL`.
 
+SQL is the language you use to communicae with the DB. Relational Databases are the actual database that can be linked to another. 
+
```

Comment:
good start. I make it a rule of thumb never to use the word I'm trying to define in the definition. If I were to expound on your statement I would say that a relational database is a management system that uses to tables to normalize data. It uses SQL -- a structured query language -- to perform CRUD operations on our API.